web/README.md

```diff
@@ -20,7 +20,15 @@ You will also see any lint errors in the console.
 
 ### `yarn lint`
 
-Lints and check for formatting problems according to [Google Typescript Style](https://github.com/google/gts).
+Lints and checks code for formatting problems according to [Google Typescript Style](https://github.com/google/gts). Lints CSS as well.
```

Comment:
should we add a link for the CSS linter also?

Reply:
@karenfrilya97 Hmmmm I'm not sure, gts comes with its set of rules (aka opinionated in some ways), but for the css linter i'm just using the standard lint rules so I'm not so sure hmm.
@ifhuang @shengdac @erencie  Any inputs on this? 😛

Reply:
Actually why do we need this stylelint? gts doesn't cover it?

Reply:
@ifhuang Nope, gts is not catered towards frontend dev specifically so CSS linter is not included! gts uses eslint and prettier behind the hood, those tools lint and format code, but not styles!

Reply:
ic. adding a link for the CSS linter SGTM